src/core/IronPython.Modules/termios.cs

```diff
@@ -19,6 +19,7 @@
 using IronPython.Runtime.Types;
 
 using static IronPython.Modules.PythonIOModule;
+using IronPython.Hosting;
```

Comment:
Not sure if this is VS being helpful again or if it's actually needed?

Reply:
No idea where it came from (or why). Funny how all those artificially intelligent tools are "clever" enough to automatically add extra `using` clauses, but not clever enough to remove them when not needed after all or even to keep them sorted.